assets/models/system/phi-3-mini-4k-instruct/spec.yaml

```diff
@@ -13,6 +13,7 @@ tags:
   Featured: ""
   Preview: ""
   maas-inference: true
+  maas-finetuning-test: true
```

Comment:
Can you please confirm this won't show up FT UX for 3p users and is applicable to only internal MSFT.

Reply:
Confirmed with Julian
![image](https://github.com/Azure/azureml-assets/assets/25394373/4e08d6a9-6032-43b1-97a7-dc8b6494d1ef)